src/main/java/Warehouse/WarehouseManager/config/KafkaTopicConfig.java

```diff
@@ -0,0 +1,34 @@
+package Warehouse.WarehouseManager.config;
+
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.AdminClientConfig;
+import org.apache.kafka.clients.admin.NewTopic;
+import org.apache.kafka.common.internals.Topic;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.config.TopicBuilder;
+import org.springframework.kafka.core.KafkaAdmin;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Configuration
+public class KafkaTopicConfig {
+
+
+    @Value(value = "${spring.kafka.bootstrap-servers}")
+    private String boostrapAddress;
+
+    @Bean
+    public KafkaAdmin kafkaAdmin(){
+        Map<String, Object> configs = new HashMap<>();
+        configs.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, boostrapAddress);
+        return new KafkaAdmin(configs);
+    }
+
+    @Bean NewTopic topic(){
```

Comment:
I would move topic creation to some .sh script that is being executed against dockerized kafka + change the name to more meaningfull